src/TraceEvent/Computers/SampleProfilerThreadTimeComputer.cs

```diff
@@ -48,6 +48,12 @@ public SampleProfilerThreadTimeComputer(TraceLog eventLog, SymbolReader symbolRe
         /// </summary>
         public bool GroupByStartStopActivity;
 
+        /// <summary>
+        /// Reduce nested application insights requests by using related activity id.
+        /// </summary>
+        /// <value></value>
+        public bool ReduceNestedApplicationInsightsRequests { get; set; } = true;
```

Comment:
@xiaomi7732 I'm not sure I understand the name "Reduce" here.  Looks like you use the name ignoreApplicationInsightsRequestsWithRelatedActivityId also which makes it more clear to me what's happening.  Can we use IgnoreApplicationInsightsRequestsWithRelatedActivityId instead for the property name?